gef.py

```diff
@@ -8848,6 +8848,60 @@ def do_invoke(self, argv: List[str]) -> None:
             err("No address mapping information found")
             return
 
+        # Parse args:
+        #  - `-a` / `--addr`:
+        #    - filter by address -> parses the next arg as an int or asks gdb
+        #    the value
+        #  - `-n` / `--name`:
+        #    - filter based on section name
+        #  - If nothing is specified, print a warning and guess the type
+        current_type = 0 # 0:None, 1:Address, 2:Name
```

Comment:
Python has `enum` for this, you can make a scoped `enum` class for those values to make it more readable

Reply:
I didn't know if scoped enums were something considered cleaner or not ^^
I'll change this :)